sensor/kubernetes/fake/fake.go

```diff
@@ -61,12 +65,12 @@ func (c *clientSetImpl) Kubernetes() kubernetes.Interface {
 
 // OpenshiftApps returns nil for the openshift client for config
 func (c *clientSetImpl) OpenshiftApps() appVersioned.Interface {
-	return nil
+	return c.openshiftApps
 }
 
 // OpenshiftConfig returns nil for the openshift client for apps
```

Comment:
This comment is not accurate anymore, now that it returns an actual client, right?

Reply:
Fixed